annotation/core/models.py

```diff
@@ -12,26 +12,41 @@ def __str__(self):
         return self.user.username + " " + str(self.is_turker)
 
 
-class Tag(models.Model):
-    name = models.CharField(max_length=100)
-    text = models.CharField(max_length=100)
-    human = models.BooleanField()
+class System(models.Model):
+    """The model that generated a given Generation."""
+    name = models.CharField(max_length=200)
+    description = models.TextField()
 
-    def __str__(self):
-        return self.text
+    # @kirubarajan: is this for training data or fine-tuning?
+    finetuned_dataset_location = models.URLField(null=True)
+
+
+class Dataset(models.Model):
+    """Dataset that we sample prompts/true human continuations from."""
+    name = models.CharField(max_length=128)
+    split = models.CharField(max_length=16)
 
 
 class Prompt(models.Model):
     """Human written sentence that gets continued."""
     body = models.TextField()
```

Comment:
Shouldn't this be a list of strings with all of the human-written sentences in this?

Reply:
We won't really use that data anywhere, and we can add that kind of data in once we do need it. This model exists just to link multiple models/generations that are sampled from the same collection of human prompts.